### PR TITLE
Add metharme information

### PR DESCRIPTION
**Pull request by AlexKraak**

### CODE REVIEW

File: docs/machines/metharme.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+## Where to find
+* 136.206.15.27
+* 136.206.15.56
+* 136.206.15.61
+* 136.206.15.62
+* 136.206.15.70
+
+## Services
+* Acts as a web server
+* Has MySql server
```
Comment (butlerx) [inline docs/machines/metharme.md]: can you make a link to mysql entry

File: docs/machines/metharme.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Where to find
+* 136.206.15.27
+* 136.206.15.56
+* 136.206.15.61
+* 136.206.15.62
+* 136.206.15.70
+
+## Services
+* Acts as a web server
```
Comment (butlerx) [inline docs/machines/metharme.md]: add link to apache entry

Comment (AlexKraak) [inline docs/machines/metharme.md]: This one? http://redbrick.readthedocs.io/en/latest/web/apache24/ should there be something here?

Comment (butlerx) [inline docs/machines/metharme.md]: yes that one it needs to be written but link to it now and we can write later

**Pull request merged**